src/transport.ts

```diff
@@ -120,4 +124,43 @@ export class WebRTCTransport implements Transport, Initializable {
       remotePeer: theirPeerId,
     });
   }
+
+  private generateNoisePrologue(pc: RTCPeerConnection, ma: Multiaddr): Uint8Array {
+    let remoteCerthash = sdp.getCerthashFromMultiaddr(ma);
+    if (!remoteCerthash) {
+      throw new InvalidArgumentError('no remote tls fingerprint in multiaddr');
```

Comment:
There's an error specific to sending in a multiaddr that's inappropriate for this transport.